### PR TITLE
[REF] easy_my_coop -> cooperator

### PR DESCRIPTION
**Pull request by robinkeunen**
cf https://github.com/coopiteasy/vertical-cooperative/pull/202

### CODE REVIEW

**Review by carmenbianca — COMMENTED** [review]
This is almost impossible to review. I can't know whether you've caught everything, and false positives are hard to spot among the sea of changes.

**Comment by carmenbianca** [comment]
:tada: the CI caught a false negative!

**Review by carmenbianca — APPROVED** [review]

**Comment by carmenbianca** [comment]
Revert 8b10664 before merging

File: investor_wallet_platform_base/__manifest__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "depends": [
         "cooperator",
         "easy_my_coop_loan",
-        "easy_my_coop_api",
+        "cooperator_api",
```
Comment (robinkeunen) [inline investor_wallet_platform_base/__manifest__.py:13]: @carmenbianca do not forget to mind for dependencies in other repositories (mainly this one and obeesdoo)

**Comment by carmenbianca** [comment]
bot is being weird. doing this manually.

**Comment by carmenbianca** [comment]
Merged into 12.0-komunigi

**Comment by victor-champonnois** [comment]
@carmenbianca I think the bot don't like it when you try to merge on another branch than 12.0

**Pull request closed without merging**